cdap-cli/src/main/java/co/cask/cdap/cli/CLIConfig.java

```diff
@@ -77,6 +80,27 @@ public Integer get() {
     }
   };
 
+  /*
+   * Wrapper class for reading/writing of username + accessToken
+   */
+  private class UserAccessToken {
```

Comment:
We have control over it here: https://github.com/caskdata/cdap-clients/tree/develop/cdap-authentication-clients/java.